python/amici/petab_import.py

```diff
@@ -73,25 +73,60 @@ def _add_global_parameter(sbml_model: libsbml.Model,
 
 
 def get_fixed_parameters(
-        sbml_model: 'libsbml.Model',
-        condition_df: Optional[pd.DataFrame] = None,
+        petab_problem: petab.Problem
 ) -> List[str]:
     """
     Determine, set and return fixed model parameters.
 
-    Parameters specified in `condition_df` are turned into constants.
+    Non-estimated parameters and parameters specified in the condition table
+    are turned into constants (unless they are overridden).
     Only global SBML parameters are considered. Local parameters are ignored.
```

Comment:
we are going to use the same function for other models aswell right? maybe state this comment only applies to SBML models?